db/sql/0_msar.sql

```diff
@@ -2087,18 +2195,18 @@ $$ LANGUAGE plpgsql RETURNS NULL ON NULL INPUT;
 
 CREATE OR REPLACE FUNCTION
 msar.create_many_to_one_link(
-  from_rel_id oid,
-  to_rel_id oid,
+  frel_id oid,
```

Comment:
I prefer the `from_rel_id` better as it informs the direction of the link. Any reason for renaming it?

Reply:
Well, it was backwards. I.e., the foreign key was _from_ `to_rel_id` to `from_rel_id`. The reason I changed it was to match up with what's in the `pg_attribute` table, and related terminology. I haven't actually found a good name, so I figure it makes most sense to match up with what is in the PG catalog.

Reply:
I added more in the docstring to that effect.